tests/helpers/test_httpx_client.py

```diff
@@ -0,0 +1,143 @@
+"""Test the httpx client helper."""
+
+import httpx
+import pytest
+
+from homeassistant.core import EVENT_HOMEASSISTANT_CLOSE
+import homeassistant.helpers.httpx_client as client
+
+from tests.async_mock import Mock, patch
+
+
+async def test_async_get_async_client_with_ssl(hass):
```

Comment:
Maybe drop the first async in the name? There's no risk for confusion.

Reply:
done